### ISSUE TITLE
AgendaFactory

### ISSUE THREAD

**Issue by Datalift** [Takox/SuperAgenda#2]
I don't see how this class interacts with the rest of the rest of the source.
Could you explain?

**Comment by FranGambero** [comment]
Yes of course. AgendaFactory works as the CRUD, being in charge of adding, updating, deleting or just showing the people in an agenda, which is an ArrayList. So, it's just part of the model of the whole program; the servlets are the controlleres and the JSPs the views. 

I hope I've helped you.
